src/styles/variables.ts

```diff
@@ -112,6 +115,9 @@ export default {
     lineHeightSizeh1: getValueUsingPixelRatio(28, 32),
     lineHeightSizeh2: getValueUsingPixelRatio(24, 28),
     lineHeightSignInHeroXSmall: getValueUsingPixelRatio(32, 37),
+    lineHeightComment: 23,
```

Comment:
The value itself is a magic number, to add a little bit of a context to why this change is made - when we increase `fontSize` for emojis, the issue with variable line height for multiline text with emojis (also on current main) was much more noticeable + in some cases emojis were touching/overlapping with line above. I've increased `lineHeight` a little bit for comments both with and without emojis to have consistent UI. I went with 23px because it made the issue far less noticeable and at the same time it's not drastic change on UI.
Example from current main - notice 2nd comment from the top and how spacing between 1st and 2nd line is higher than the rest of the same comment. When we make emojis bit bigger than the rest of the text, this spacing increases and is much more noticeable 
<img width="559" alt="main_ios_workspace" src="https://github.com/Expensify/App/assets/26638539/ead13d56-23cd-46fd-9a79-397c3b7834e6">